rollup.config.js

```diff
@@ -65,6 +67,6 @@ export default [
       globals,
     },
     external,
-    plugins: [babel(babelOptions()), resolve(), commonjs(), uglify({}, minify)],
+    plugins: [babel(babelOptions()), resolve(), commonjs(), terser()],
```

Comment:
Add `include: /node_modules/` to commonjs plugin to skip parsing for source files.